ts/utils/payment.ts

```diff
@@ -320,6 +320,8 @@ export const getV2ErrorMainType = (
     case "PAA_PAGAMENTO_DUPLICATO":
     case "PPT_PAGAMENTO_DUPLICATO":
       return "DUPLICATED";
+    case "PAA_PAGAMENTO_SCONOSCIUTO":
```

Comment:
As mentioned in the JIRA ticket, we have to cover both `PAA_PAGAMENTO_SCONOSCIUTO` and `PPT_PAGAMENTO_SCONOSCIUTO`

Reply:
Whoops, I forgot it :) Fixing asap

Reply:
After some investigation with the BE team it turned out that `PPT_PAGAMENTO_SCONOSCIUTO` can not reach this stage of payment/verification, so it can be omitted.